src/compositemark/errorbar.ts

```diff
@@ -340,10 +349,29 @@ export function errorBarParams<
     encodingWithoutContinuousAxis,
     ticksOrient: orient === 'vertical' ? 'horizontal' : 'vertical',
     markDef,
-    outerSpec
+    outerSpec,
+    tooltipEncoding
   };
 }
 
+export function errorBarTooltipEncoding(
+  continuousAxisChannelDef: PositionFieldDef<string>,
+  encodingWithoutContinuousAxis: Encoding<string>
+): Encoding<string> {
+  const tooltopSummaryMap: CompositeMarkTooltipSummary[] = [
+    {fieldPrefix: 'upper', titlePrefix: 'Upper error'},
+    {fieldPrefix: 'lower', titlePrefix: 'Lower error'}
+  ];
+
+  const tooltip: TextFieldDef<string>[] = getCompositeMarkTooltip(
```

Comment:
Why shouldn't `getCompositeMarkTooltip` return the output wrapped with `{tooltip: ...}` already? 

Also, I wonder if `errorBarTooltipEncoding` needs to be a separate method at all?   (Same for box)

It seems like you only need to define `errorBarTooltipSummaries()` (I think the name `-Map` is confusing) and just call `getCompositeMarkTooltip` directly.